src/supertux/main.cpp

```diff
@@ -202,7 +202,7 @@ class PhysfsSubsystem
     }
     else
     {
-		userdir = PHYSFS_getPrefDir("SuperTux","supertux2");
+		userdir = PHYSFS_getPrefDir("SuperTux", PACKAGE_NAME);
```

Comment:
Can we get "SuperTux" a separately defined constant too? I imagine it could be useful, also in the future (e.g. for scripts that run via CMake).